### ISSUE TITLE
Make disabled buttons accessible

### ISSUE THREAD

**Issue by jeremyckahn** [jeremyckahn/farmhand#273]
From #272:

> Disabling buttons in this type of way isn't actually accessible. It ends up "removing" them from the DOM tree and then screen readers don't know they're there. Long term, disabling buttons should be visual only, and we have to add logic to the click handlers to prevent the action when it's not allowed. i went with the disabled route for this PR to match how the other areas of the UI are currently working and patch the hole for the majority to start. accessibility can be a separate effort since we know there are other issues here and there.
